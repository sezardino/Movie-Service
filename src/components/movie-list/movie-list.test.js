import React from 'react';
import renderer from 'react-test-renderer';
import MovieList from './movie-list.jsx';

it(`MovieList are rendered`, () => {
	const tree = renderer.create(<MovieList movies={[]} />).toJSON();

	expect(tree).toMatchSnapshot();
});
