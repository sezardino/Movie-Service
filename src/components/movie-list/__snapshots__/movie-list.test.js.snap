// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieList are rendered 1`] = `
Array [
  <div
    className="catalog__movies-list"
  />,
  <div
    className="catalog__more"
  >
    <button
      className="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>,
]
`;
